tests/kernel/install_ltp.pm

```diff
@@ -281,7 +281,7 @@ sub run {
     upload_runtest_files('${LTPROOT:-/opt/ltp}/runtest', $tag);
 
     select_console('root-console');
-    #type_string "reboot\n";
+    type_string "reboot\n" if get_var('LTP_INSTALL_REBOOT');
```

Comment:
Have you considered using power_action? I know it's not directly related to the change you propose, but this won't work on zVM, for example, or ipmi. And power_action has all the tweaks for different setups.

Reply:
ok, thanks

Reply:
Just give it a try, if it doesn't work straight away, let's merge this PR and fix it later. Cheers!